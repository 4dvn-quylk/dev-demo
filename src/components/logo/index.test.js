/* eslint-disable testing-library/prefer-screen-queries */
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import { Logo } from './index';

describe('Test <Logo /> component', () => {
  test('should render', () => {
    const { container } = render(<Logo />);
    const logo = screen.getByTestId('c-logo');
    expect(logo).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
