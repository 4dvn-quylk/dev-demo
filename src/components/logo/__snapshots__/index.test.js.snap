// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Logo /> component should render 1`] = `
<div>
  <span
    class="ant-avatar ant-avatar-circle ant-avatar-image"
    data-testid="c-logo"
    style="width: 64px; height: 64px; line-height: 64px; font-size: 18px;"
  >
    <img
      alt="header-logo"
      src="logo.svg"
    />
  </span>
</div>
`;
